### PR TITLE
Improve docker build time

### PR DESCRIPTION
**Pull request by mzottola**

### CODE REVIEW

File: src/container_registry/docker.rs [hunk 1]
```diff
@@ -189,6 +190,59 @@ pub fn docker_tag_and_push_image(
             info!("image {} has successfully been pushed", image_with_tag);
             Ok(())
         }
+    };
+
+    let image_with_latest_tag = format!("{}:latest", image_name);
```
Comment (erebe) [inline src/container_registry/docker.rs:195]: re-use name_with_latest_tag()

**Review by erebe — DISMISSED** [review]

File: src/container_registry/docker.rs (hunk 1 above)
Comment (mzottola) [reply src/container_registry/docker.rs]: done

File: src/transaction.rs [hunk 2]
```diff
@@ -125,44 +125,28 @@ impl<'a> Transaction<'a> {
...
+                        "something goes wrong while pulling image from {:?} container registry",
+                        container_registry.kind()
+                    ))
+                );
+                return Err(EngineError::new_from_legacy_engine_error(err));
             }
         };
 
-        Ok(())
+        return Ok(());
```
Comment (benjaminch) [inline src/transaction.rs:149]: OPT: no need for explicit `return` can be `Ok(())` without `;`

**Review by benjaminch — DISMISSED** [review]

File: src/transaction.rs (hunk 2 above)
Comment (mzottola) [reply src/transaction.rs]: done

**Review by benjaminch — APPROVED** [review]

**Review by erebe — APPROVED** [review]

**Pull request merged**